pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -71,14 +73,35 @@ func (ds *dockerService) determinePodIPBySandboxID(sandboxID string) string {
 		if err != nil {
 			continue
 		}
-		if containerIP := getContainerIP(r); containerIP != "" {
-			return containerIP
+
+		// On Windows, every container that is created in a Sandbox, needs to invoke CNI plugin again for adding the Network,
+		// with the shared container name as NetNS info,
```

Comment:
This comment doesn't square with getNetworkNamespace() below, which appears to return the network mode (not a shared container name) as the NetNS info.  Or am I reading it wrong?

Reply:
Yes, it returns the networkmode, which is container:<shared containername>